core/toolbox/toolbox.js

```diff
@@ -155,6 +154,13 @@ Blockly.Toolbox = function(workspace) {
   this.boundEvents_ = [];
 };
 
+/**
+ * Decides how a cursor should act when it is on the toolbox.
+ * @type {function(!Blockly.ShortcutRegistry.KeyboardShortcut):boolean}
+ * @public
+ */
+Blockly.Toolbox.prototype.onShortcut;
```

Comment:
Since `onShortcut` isn't implemented, doesn't that mean that this class isn't properly implementing the `IKeyboardAccessible ` interface?